core/new-gui/src/app/workspace/component/menu/menu.component.html

```diff
@@ -3,6 +3,7 @@
     <a href="/">
       <img
         alt="Texera"
+        height="{{userSystemEnabled?75:38}}"
```

Comment:
it seems that 75 and 38 are hard coded. just want to make sure there is no relative way to do it? For example I thought the logo should just be the same height of its parent (i.e., the menu bar)?

Reply:
It's due to the limitation of flex block in css. The height of the parent block(menu bar) will depend on this height.